src/App/Hovedside/AltinnContainer/AltinnContainer.tsx

```diff
@@ -120,10 +110,11 @@ const AltinnContainer: FunctionComponent = () => {
               alt="ikon for å beskrive at lenken åpnes i en ny fane"
             />
           </Lenkepanel>
-        )}
+        </div>
       </div>
-    </div>
-  );
+    );
+  }
+  return null;
 };
 
 export default AltinnContainer;
```

Comment:
Det er ganske mye gjntagende kode i denne filen, kanskje man kunne lagd en slaks wrapper dings rundt <lenkerpanel> som kun tar inn det som er forskjellig for hver enkelt <lenkepanel>